app/src/main/java/com/kimdoori/githubapp/viewcommon/BaseAdapter.kt

```diff
@@ -0,0 +1,25 @@
+package com.kimdoori.githubapp.viewcommon
+
+import androidx.recyclerview.widget.RecyclerView
+
+abstract class BaseAdapter<ITEM_TYPE : Any> : RecyclerView.Adapter<BaseViewHolder<ITEM_TYPE>>() {
```

Comment:
```suggestion
abstract class BaseAdapter<T> : RecyclerView.Adapter<BaseViewHolder<T>>() {
```

제네릭은 통상적으로 타입이 하나인 경우에는 T 로만 작성하고 어떤 타입인지 생략합니다.
만약에 제네릭이 두 개 이상인 경우에도 알파벳 하나로만 작성하는 경우가 있지만
가독성을 위해 타입을 추가해주면 좋긴 합니다 😄

Reply:
https://github.com/kimdoori/GitHubApp/pull/18/commits/9994b94c66e1eb3ca6f67565b4fc00228e8924e7 T로 변경했습니다! 이 커밋에 BaseAdapter, BaseViewHolder 관련 리뷰들이 같이 있습니다!